01-fundamentals/first_letter.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+def get_user_password():
+    password = input("Tell me your password:")
```

Comment:
Just a nitpick: add a space after the colon for better "UX".
(`Tell me your password:12345` vs `Tell me your password: 12345`)